js/build.js

```diff
@@ -24,20 +24,20 @@ Fliplet.Widget.instance({
     beforeReady: function() {
       let element = $(this.$el);
 
-      Fliplet.Env.get('interact')
-        ? element.addClass('edit') : element.removeClass('edit');
+      element.toggleClass('edit', Fliplet.Env.get('interact'));
     },
     ready: async function() {
       await Fliplet.Widget.initializeChildren(this.$el, this);
 
-      function ifArray(elem) {
+      var result;
+      const userNotLoggedMessage = 'User is not logged in';
+
+      function getType(elem) {
         if (Array.isArray(elem)) {
           return 'array';
-        } else if (typeof elem === 'string') {
-          return 'string';
         }
 
-        return false;
+        return typeof elem === 'string' ? 'string' : false;
```

Comment:
Just return the type and have the function do what it's named to do. You're only checking for the `array` and `string` results anyway.

```suggestion
        return typeof elem;
```